plugins/cilium-cni/chaining/terway/terway.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package terway
+
+import (
+	chainingapi "github.com/cilium/cilium/plugins/cilium-cni/chaining/api"
+	"github.com/cilium/cilium/plugins/cilium-cni/chaining/pod-link"
+)
+
+func init() {
+	chainingapi.Register("terway-chainer", &podlink.GenericLink{})
```

Comment:
Probably this can be even a generic chainner of devices which are in a pod netns and don't have any pair in a host netns ("pod-netns-chainer"?).

Reply:
Yes. This pr has contained a pod-link chainer in ```plugins/cilium-cni/chaining/pod-link```. Do you mean this?

Reply:
Aha, I see. Any reason for introducing "terway-chainer"? Couldn't user just use the "pod-link" chainer?

Reply:
yeah, I will only use the 'pod-link' one.